Algo fair (#179)
very-good-science/data-ethics-club: site/write_ups/2022/05-10-22_writeup.md

# Data Ethics Club: [The Failures of Algorithmic Fairness](https://senseoffairness.blog/2020/12/17/the-failures-of-algorithmic-fairness/))
<!--Please don't edit the info panel below-->

```{admonition} What's this? 
This is summary of Wednesday 5th October's Data Ethics Club discussion, where we spoke and wrote about the [The Failures of Algorithmic Fairness](https://senseoffairness.blog/2020/12/17/the-failures-of-algorithmic-fairness/), a blog post by Paul Lee.
The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
Nina Di Cara and Natalie Thurlby helped with the final edit.
```

## Discussion

```{image} images/AlgoFair.png
:width: 200px
:align: center
:alt: Gru meme. First panel reads: "Recruiters make an algorithm to rank applicants for particular jobs". Second panel reads: "Applicants work out which proxies to optimise for". Third and fourth panel read: "Algorithm eventually ranks all candidates the same", this makes Gru sad. Meme made by Huw Day.
```

### Are impacts of fairnesses always fair? 

> Some of our reactions to this piece were "we agree, and now what?" We keep reading these sorts of articles looking for an answer but still no one has solved algorithmic unfairness. Similar question to what we've looked at before: why are we doing all this work to gather more and more information, will it actually solve anything? Human-created problems need human-created solutions.

Reviewer: ```suggestion
Some of our reactions to this piece were "we agree, and now what?" We keep reading these sorts of articles looking for an answer but still no one has solved algorithmic unfairness. 
It's a similar question to what we've looked at before: why are we doing all this work to gather more and more information, will it actually solve anything? Human-created problems need human-created solutions.
```

The article seems to shy away from making any strong statements. Using proxies in a black box way, even with protected variables say for example with insurances. The unfairness of certain algorithms is the point. The article could've pushed a bit more towards this. The article is very accessible and not aimed at subject matter experts, but occassionally generalises a bit too much: e.g. failing to make a distinction between an AI model and a hand written algorithm, for example. It feels like there is a bit of a push for people to comment on AI and unfairness in this area. 

> We spotted an error where the picture shown for the cameras is about false negatives not false positives. The probability of getting away with the crime. 

Reviewer: ```suggestion
```
I think this is one of the bits which actually wasn't an error in the end?

There were divided opinions on the subject of the A-level results algorithm. Many of us felt that if you get some kind of automated system to assign A-level grades without proper input from humans then on day one, would it just give everyone their grades at birth based on postcode? Adding algorithms into unfair systems just encodes the unfairness while pretending it's objective.

> If you are improving over years, that was accounted for predicted and taken into account or if you got better results in GCSE results. Exam results are always biased by class and that wasn't increased by the algorithm. Some argued that there are a fixed number of places at universities and that A-level grades are essentially used to assign these places. We had a brief aside to question: are these grades good at predicting people's success in universities? Are the critiques just with how exams assess us at all? If you automate a flawed system, it will remain flawed and become systematically so. Perhaps more efforts should be made to understand *why* and *how* this particular system was flawed to give arguments (whether they be for or against) more weight.

Reviewer: ```suggestion
If you are improving over years, that was accounted for in the prediction, or if you got better results in GCSE. Exam results are always biased by class and that wasn't increased by the algorithm. Some argued that there are a fixed number of places at universities and that A-level grades are essentially used to assign these places. We had a brief aside to question: are these grades good at predicting people's success in universities? Are the critiques just with how exams assess us at all? If you automate a flawed system, it will remain flawed and become systematically so. Perhaps more efforts should be made to understand *why* and *how* this particular system was flawed to give arguments (whether they be for or against) more weight.
```
